core/views.py

```diff
@@ -583,3 +589,41 @@ def get_context_data(self, **kwargs):
         ctx['results'] = results
 
         return ctx
+
+class OpenJobListView(APIView):
+    def get(self, request):
+        # Parameters recieving
+        limit = int(request.query_params.get('limit', 10))
+        offset = int(request.query_params.get('offset', 0))
+        base_url = settings.JOBS_SCRAPE_BASE_URL
+        url = base_url + "/search"
+
+        try:
+            response = requests.get(url, timeout=30)
```

Comment:
We may want to lower this. When you specify the timeout this way, it's going to have a connection timeout of 30 seconds, then an additional 30 seconds to receive the data, so we're looking at a full 60 seconds that a request could be waiting. I would go ahead and lower this down to `10`, which gives a very reasonable 10 seconds to connection (which is 3 tries using the standard 3 second retry interval for tcp/ip requests) and 10 seconds between bytes sent in case the connection is getting interrupted.

More reading here if you're interested in how this setting works: https://requests.readthedocs.io/en/latest/user/advanced/#timeouts

Reply:
Thank you for sharing the documentation! it is really informative!